ml/Dimension.h

```diff
@@ -88,6 +88,8 @@ class PredictableDimension : public TrainableDimension {
 
     bool isAnomalous() { return AnomalyBit; }
 
+    double AnomalousBitCount;
```

Comment:
Default-initialize this with:

`double AnomalousBitCount{0.0};`

Also, it would be better to make it a private variable and add public getter/setter functions in order to be consistent with the rest of the member variables of the class.